velox/substrait/tests/FunctionTest.cpp

```diff
@@ -40,8 +40,18 @@ class FunctionTest : public ::testing::Test {
   std::shared_ptr<vestrait::SubstraitParser> substraitParser_ =
       std::make_shared<vestrait::SubstraitParser>();
 
+  std::shared_ptr<ConnectorHandler> getConnector() {
+    return std::make_shared<HiveConnectorHandler>(
+        "function-test-hive",
+        "function-hive-table",
```

Comment:
It is a bit strange to hard-code table name here. Shouldn't it come from the Substrait plan? How would we process queries that read from multiple tables (union, join)?